lib/comply/whitelist_constantize.rb

```diff
@@ -0,0 +1,30 @@
+require 'active_model/validations'
+
+module Comply
+  module WhitelistConstantize
+    class NotWhitelistedError < ArgumentError
+    end
+
+    DEFAULT_PROC = -> (klass) { klass <= ActiveModel::Validations }
+
+    class << self
+      def constantize(str)
+        whitelist! { str.classify.constantize }
+      end
+
+      def whitelist
+        @whitelist ||= [DEFAULT_PROC]
+      end
+
+      private
+
+      def whitelist!
+        yield.tap do |klass|
+          whitelist.detect(-> { raise NotWhitelistedError }) do |matcher|
```

Comment:
I had no idea until now that detect accepts an `ifnone` argument - could you add a comment?